index.rst

```diff
@@ -6,24 +6,233 @@
+This was difficult since it was incompatible with both the balancing procedure and the balancing equipment.
+The procedure requires balancing the y-axis at zenith pointing and the z-axis at horizon pointing. The axes are balanced by driving motorized balancing units.
+The units are interlocked so then cannot be operated other than horizon pointing or zenith pointing, when the elevation pins are inserted.
+
+
+The :ref:`figure below <after-balancing>` shows the updated measured torque profiles for the elevation range 5-90 *deg*. This does not include the problematic region of 0-5 *deg*.  We show slew profiles at two different velocity configurations, 1% (0.05 *deg/s*) and 10% (0.5 *deg/s*) of the designed max velocity (5.25 *deg/s* TLS-REQ-0159). Upward slews are shown in orange and downward slews in blue. It can be seen that for both configurations the majority of the observed variation in required torque as a function of elevation is gone. The torque hysteresis appears to be reduced to ~ 4 *kNm* (+/- 2 *kNm*) and have little dependence on the speed of the slew.
+
+.. figure:: ./_static/elevation_slews_after_balancing_20230630.png
+   :name: after-balancing
+
+   Each panel shows required torque as a function of elevation for slews between 0 and 90 *deg*. Upward slews are shown in orange and downward slews in blue, the lighter shaded area shows raw measured values, and the darker line denotes a rolling mean. The top (bottom) panel shows a pair of slews at 1% (10%) of the designed maximum velocity.
+
+Next, we show a comparison of the torque profiles before and after masking the 0-5 *deg* region. This figure makes it clear that the majority of the previously observed hysteresis was due to trying to account for the heterogenously behaved low elevation region. All slews are run at 1% velocity configuration.
+
+.. figure:: ./_static/elevation_slews_comparison_20230630.png
+   :name: compare-slews
+
+   A comparison of required torque as a function of elevation. This demonstrates the improvement in hysteresis gained by balancing the telescope while masking out the region from 0-5 *deg*
+
+Disabled Elevation Drives
+=========================
+According to the EUI Elevation drives 31 and 32, figure 9, are not enabled when operating, figure 10.
+The EUI shows Elevation drives 31 and 32 disabled.
+Notice on the pictures all the other drives are enabled when 31 and 32 are disabled.
+It is possible that the logic is inverted.
+Because when all other drives are enabled, 31 and 32 are disabled.
+Note: The drives are mislabeled. Drives 31 and 32 are physically on +X side, but according to the EUI they are on the -X side.
+
+.. image:: _static/image_of_drive_31_32.png
+   :width: 49 %
+.. image:: _static/eui_image.png
+   :width: 49 %
+
+Left image: Drives 31 and 32 on the +X axis
+
+Right image: EUI screenshot showing drives 31 and 32 enabled when the rest are disabled. The opposite behaviour is also observed (31,32 disabled when the rest are enabled)
```

Comment:
All this should be part of the figure caption. Just remove the blank line and use indentation to make it consistent with the rest of the document (caption with indentation).